doc/Docker_deployment.md

```diff
@@ -0,0 +1,29 @@
+# Deploying a module in a local Docker development environment
+
+## Build your module
+
+See [Module_builder.md](Module_builder.md).
+
+## Obtain prerequisites
+
+1. Docker Toolbox: https://www.docker.com/toolbox
+
+## Building base developer environment
+
+Note: this has only been tested in OS X.
+
+2. Run Kitematic in default configuration (this will start up a VirtualBox machine called "default")
+3. Open the "Docker CLI" (button in lower-left window)
+4. Run these commands in the terminal (these are to work around an issue with using private docker registries inside a VirtualBox VM):
+
+        VBoxManage modifyvm "default" --natdnshostresolver1 on
+        VBoxManage modifyvm "default --natdnsproxy1 on
+6. Exit terminal
+7. Quit and restart Kitematic
+8. Open Docker CLI
+9. docker pull kbase/deplbase
```

Comment:
We're not sure which Docker image to use yet.  The deplbase image needs a copy of the cluster.ini file and an ssl directory that is created by deploy_dev.  But it's the only image we have publicly available right now.